### PR TITLE
core: enable error applet, add stubs for web applet

### PR DESCRIPTION
**Pull request by liamwhite**
This adds support for and enables the error applet by default, and adds preliminary support for the online web applet.
|Error|Web|
|-|-|
||0100ea80032ea000_2024-02-24_22-36-40-756|

### CODE REVIEW

File: src/core/hle/service/ssl/cert_store.cpp [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            const auto entry = m_certs.find(certificate_id);
+            if (entry == m_certs.end()) {
+                continue;
+            }
+            f(*entry);
+        }
+    }
+}
+
+Result CertStore::GetCertifcates(u32* out_num_entries, std::span<u8> out_data,
```
Comment (FearlessTobi) [inline src/core/hle/service/ssl/cert_store.cpp:95]: spelling error?

File: src/core/hle/service/ssl/ssl_types.h [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+enum class TrustedCertStatus : s32 {
+    Invalid = -1,
+    Removed = 0,
+    EnabledTrusted = 1,
+    EnabledNotTrusted = 2,
+    Revoked = 3,
+};
+
+struct BuiltInCertificateInfo {
```
Comment (FearlessTobi) [inline src/core/hle/service/ssl/ssl_types.h:85]: size assert, same below

**Review by FearlessTobi — APPROVED** [review]

**Comment by FearlessTobi** [comment]
> ssl/cert_store.cpp:122:24: error: conversion from ‘std::vector<unsigned char>::size_type’ {aka ‘long long unsigned int’} to ‘u32’ {aka ‘unsigned int’} may change value [-Werror=conversion]
  122 |         cur_der_offset += cur_der_data.size();

**Comment by liamwhite** [comment]
So after some checking the online web applet is actually functional, just not when nsd has to be used

**Pull request merged**